composer.json

```
@@ -1,7 +1,7 @@
 {
     "name": "dotkernel/dot-twigrenderer",
     "type": "library",
-    "description": "DotKernel component providing twig extensions and customization",
+    "description": "Dotkernel component providing twig extensions and customization",
     "license": "MIT",
         "keywords": [
```

```suggestion
    "keywords": [
```
Decrease indent on `keywords` so it aligns with the other keys.